src/NuGet.Core/NuGet.Commands/PublicAPI/net472/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,7 @@
 #nullable enable
+NuGet.Commands.ConfigPathsArgs
```

Comment:
Given that this is only going to be implement in `dotnet nuget`, should we move this to the NuGet.CommandLine.Xplat assembly?

cc @NuGet/nuget-client thoughts?

Reply:
When we we're looking at other `dotnet nuget` commands, we saw that we were using autogenerated files using `dotnet-t4 global tool` (used in dotnet nuget ... source https://github.com/NuGet/NuGet.Client/pull/3206), so since these are files are autogenerated I suggest not moving them if we plan to continue using the tool. 

These files are a lot of boilerplates and I think if we move this code to `NuGet.CommandLine.Xplat` it would work the same, but I think it's more a discussion of if we should continue using the tool or manually create the files. cc: @NuGet/nuget-client

Reply:
Do we absolutely have to move all the files?

Reply:
The only one is not autogenerated is `NuGet.Commands.ConfigPathsRunner`, so we can move that one

Reply:
We ended up moving three files to the Commands folder in `NuGet.CommandLine.XPlat` and are no longer using the tool to autogenerate code.